### ISSUE TITLE
Tests fail for non-US contributors

### ISSUE THREAD

**Issue by CAYdenberg** [moment/luxon#110]
Steps to reproduce:

Clone the repo, install all depedencies, install full-icu and run tests using `./scripts/test`

There are 10 failed tests. One of them (**for me**) is:

```bash
  ● DateTime#toObject accepts a flag to return config

    expect(received).toEqual(expected)
    
    Expected value to equal:
      {"day": 25, "hour": 9, "locale": "en-US", "millisecond": 123, "minute": 23, "month": 5, "numberingSystem": null, "outputCalendar": null, "second": 54, "year": 1982}
    Received:
      {"day": 25, "hour": 9, "locale": "en-CA", "millisecond": 123, "minute": 23, "month": 5, "numberingSystem": null, "outputCalendar": null, "second": 54, "year": 1982}
    
    Difference:
    
    - Expected
    + Received
    
    @@ -1,9 +1,9 @@
      Object {
        "day": 25,
        "hour": 9,
    -   "locale": "en-US",
    +   "locale": "en-CA",
        "millisecond": 123,
        "minute": 23,
        "month": 5,
        "numberingSystem": null,
        "outputCalendar": null,
```

If I run `LANG=en_US.utf8` first all tests pass.

It might be enough to change `./scripts/test.js` to set LANG first, but this might not work for users who have their system set up to run `gulp test` directly. Possibly Jest can be configured to set env variables before running?

**Comment by icambron** [comment]
Yeah, I should have seen that coming when I changed the locale defaulting behavior. I think I'll add the env var to that script. It's definitely possible that we could set that env var using JS in the gulpfile. But I'll go with the quick fix for now.

**Comment by icambron** [comment]
Fixed in 2fde8d5
